vars/Ubuntu_xenial.yml

```diff
@@ -0,0 +1,13 @@
+---
+grsecurity_build_apt_packages:
```

Comment:
:+1: I've had to drop in a similar vars file for Xenial support, great to have it as part of the role.